back/app/controllers/web_api/v1/comments_controller.rb

```diff
@@ -255,4 +263,16 @@ def merge_comments(root_comments, child_comments)
       [root_comment, *children_by_parent[root_comment.id]]
     end
   end
+
+  def anonymous_not_allowed?
```

Comment:
Should this be a private function?

Reply:
Actually, it already is a private method :)

Reply:
Oh sorry, must have misread the code!